src/js/controller/instream-adapter.js

```diff
@@ -194,6 +192,9 @@ var InstreamAdapter = function(_controller, _model, _view, _mediaPool) {
         if (_destroyed || !_inited) {
             return Promise.reject(new Error('Instream not setup'));
         }
+        
+        this.setText(_model.get('localization').loadingAd);
```

Comment:
`loadItem` is never called for IMA so doesn't break the change for IMA?

Reply:
This change would only set the text once a creative is loading.

The text should be set while a tag is loaded. To ensure this, calls `setText` in vast instead of  in instream.